torch/_torch_docs.py

```diff
@@ -3122,8 +3124,9 @@ def merge_dicts(*dicts):
 Example::
 
     >>> weights = torch.tensor([0, 10, 3, 0], dtype=torch.float) # create a tensor of weights
-    >>> torch.multinomial(weights, 4)
-    tensor([ 1,  2,  0,  0])
+    >>> torch.multinomial(weights, 2)
+    tensor([1, 2])
+    >>> torch.multinomial(weights, 4) # ERROR!
```

Comment:
I'd add a few words explaining the error. Adding the error message here would also be useful.